routes/courses.js

```diff
@@ -1,51 +1,133 @@
 const express = require('express');
+const { Client } = require('pg');
 const router = express.Router();
 const MOCK_COURSES = require('./mock-courses.json');
 const _ = require('lodash');
+const changeCase = require('change-case');
+
+const client = new Client({
+  user: 'postgres',
+  password: 'kong8190',
+  host: 'localhost',
+  port: 5432,
+  database: 'cip'
+});
+
+client.connect(err => {
+  if (err) {
+    throw err;
+  }
+  console.log('Postgresql connected...');
+});
 
 /*
   @route  GET /courses
   @desc   Get courses with filter and paignation
   @access public
 */
-router.get('/', (req, res) => {
+router.get('/', async (req, res) => {
   try {
     const filters = req.query.filters || {};
-    const offset = parseInt(req.query.offset) || 0;
+    const page = parseInt(req.query.offset) || 0;
     const size = parseInt(req.query.size) || 10;
+    const offset = page * size;
 
-    const parsedFilters = JSON.parse(filters);
+    let parsedFilters = JSON.parse(filters);
+
+    parsedFilters = _.each(parsedFilters, (value, key) => {
+      if (key === 'year') {
+        let startYears = [];
```

Comment:
These array can be `const` since you are not changing the array reference, but just updating its value with `push`